apps/morph_check

```diff
@@ -185,22 +186,57 @@ def test_file(f, config):
 
         result &= ok
 
-    return result, {f: summary_}
+    summary['status'] = 'PASS' if result else 'FAIL'
+
+    return result, {f: summary}
+
+
+def check_files(files, config):
+    '''Test a bunch of files'''
+
+    SEPARATOR = '=' * 40
+    summary = {}
+    res = True
+
+    for _f in files:
+        L.info(SEPARATOR)
+        try:
+            status, summ = check_file(_f, config)
+            log_msg(status, 'Check result:')
+            res &= status
+            summary.update(summ)
+        except IDSequenceError as e:
+            print 'ID ERROR in file %s: %s' % (_f, e.message)
+        except StandardError as e:
+            print 'ERROR: Could not read file %s: %s' % (_f, e.message)
+
+    L.info(SEPARATOR)
+
+    status = 'PASS' if res else 'FAIL'
+
+    return {'files': summary, 'status': status}
+
+
+# Default check configuration parameters
+CONFIG = {'neuron_checks': ['has_basal_dendrite',
+                            'has_axon',
+                            'has_apical_dendrite',
+                            'nonzero_segment_lengths',
+                            'nonzero_section_lengths',
+                            'nonzero_neurite_radii'],
+          'options': {"nonzero_neurite_radii": 0.007,
+                      "nonzero_segment_lengths": 0.01,
+                      "nonzero_section_lengths": 0.01}}
 
 
 if __name__ == '__main__':
```

Comment:
I will leave that for later. It requires fixing the logging, because there is a re-definition of `log_msg` in the current `main`.